src/Services/Files.ts

```diff
@@ -0,0 +1,68 @@
+import {AxiosResponse} from 'axios';
+import Service from './Service';
+import Xuid, {SupportedXuid} from '../Utils/Xuid';
+import JsonResponse from '../Interfaces/JsonResponse';
+import {File} from '../Models/Models';
+import PaginatedJsonResponse from '../Interfaces/PaginatedJsonResponse';
+
+/**
+ * Files Api
+ */
+class Files extends Service
+{
+
+    /**
+     * List all files
+     * @param pageNumber
+     * @returns Promise<file>
+     */
+    list(pageNumber: number = 1) {
+        return this.client
+        .get<PaginatedJsonResponse<File>>('1/files')
+        .then((response: AxiosResponse<PaginatedJsonResponse<File>>) => {
+            return response.data.data
+        });
+    }
+
+    /**
+     * Create file
+     * @param file
+     * @returns Promise<file>
+     */
+    create(file: File) {
+        return this.client
+            .post<JsonResponse<File>>(`1/filees`, file)
+            .then((response: AxiosResponse<JsonResponse<File>>) => {
+                return response.data.data;
+            });
+    }
```

Comment:
You'll need to be able to provide more parameters such as `type` and in most cases `business_id`